tests/expectations/metrics/test_metrics_util.py

```diff
@@ -354,20 +360,27 @@ def test_get_dbms_compatible_metric_domain_column_kwargs(
 
 @pytest.mark.unit
 @pytest.mark.parametrize(
-    "input_column_name_a, input_column_name_b, output_column_name_a, output_column_name_b",
     [
-        [
+        "input_column_name_a",
+        "input_column_name_b",
+        "output_column_name_a",
+        "output_column_name_b",
+    ],
+    [
+        pytest.param(
             "ARTISTS",
```

Comment:
Could even change this to something like
```suggestion
            "NOT_QUOTED",
```

Reply:
@anthonyburdi Fixed -- committed!  Thanks!